app-media-recorder.html

```diff
@@ -213,6 +214,10 @@
          * Start recording from the source media stream.
          */
         start: function() {
+          if (!supported) {
```

Comment:
these supported checks needed to be added so that it could modulize correctly. Modulizer gets rid of iifes so that it can export values nicely. This means the return statement approach fails. Additionally, putting the element definition in an `else` block messed with modulizer's namespace resolver